src/components/Scheme/SchemeContainer.tsx

```diff
@@ -257,6 +232,8 @@ const SubscribedSchemeContainer = withSubscription({
 
     return combineLatest(
       of(schemeState),
+      // Find the SchemeManager scheme if this dao has one
+      Scheme.search(arc, {where: { dao: props.daoState.id, name: "SchemeRegistrar" }}).pipe(mergeMap((scheme: Array<Scheme | CompetitionScheme>): Observable<ISchemeState> => scheme[0] ? scheme[0].state() : of(null))),
```

Comment:
why is CompetitionScheme needed here? We're looking for SchemeRegistrar.

can we document that in the code?

Reply:
its not, its just the required type